mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping

```diff
@@ -1 +1,10 @@
 CLASS net/minecraft/class_957 net/minecraft/client/render/entity/TntMinecartEntityRenderer
+	METHOD method_23190 renderFlashingBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IZ)V
+		COMMENT Renders a given block state into the given buffers either normally or with a bright white overlay.
+		COMMENT Used for rendering primed TNT either standalone or as part of a TNT minecart.
+		ARG 0 blockState
+		ARG 1 matrices
+		ARG 2 vertexConsumers
+		ARG 3 light
+		ARG 4 drawFlash
+			COMMENT If true, a white semi-transparent overlay is added to the block to indicate the flash.
```

Comment:
```suggestion
			COMMENT whether a white semi-transparent overlay is added to the block to indicate the flash
```
Note that in fabric yarn we use phrases than full sentences for javadoc tag comments like for `@param` `@return` etc